corehq/motech/repeaters/views/repeat_record_display.py

```diff
@@ -37,6 +37,8 @@ def last_checked(self):
 
     @property
     def next_check(self):
+        if self.record.state not in (RECORD_PENDING_STATE, RECORD_FAILURE_STATE):
+            return '---'
```

Comment:
I think we should move this list of "queued" states into a constant. Not sure if there is currently anywhere else that references this list of states, but I know there will be soon!